src/adapters/pnp/daemon/osconfig.json

```diff
@@ -861,5 +861,5 @@
       "ObjectName": "maxManualAttestationsPerDay"
     }
   ],
-  "ReportingIntervalSeconds": 30
+  "ReportingIntervalSeconds": 300
```

Comment:
Interesting. Is there an issue with a non-existent IoT Hub connection and shorter intervals here?

Reply:
OSConfig works in vain a lot attempting to get a conn from AIS to connect to IoT Hub. Lots of machine resources consumed for nothing.